### PR TITLE
Fixing error in ast list generating script

### LINKED ISSUE #424: Bug in argparse for make_ast_inputs.py
I get the following error when running the make asts inputs step in the standard workflow.

> (base) kgordon@icy PHAT_M31]$ python -m beast.tools.run.make_ast_inputs --flux_bin_method=True
Auto-detected type: fits
Auto-detected type: fits
usage: make_ast_inputs.py [-h] [--flux_bin_method FLUX_BIN_METHOD]
make_ast_inputs.py: error: argument --flux_bin_method: invalid int value: 'True'

This is due to the argparse parameter setup expecting an integer, not a boolean.

### PR DESCRIPTION
**Pull request by karllark**
Closes #424.

### CODE REVIEW

**Comment by lea-hagen** [comment]
I think this latest change is mixing up two different parts of the AST process.  I'll summarize the steps/options to make sure I'm clear:
1. Pick SEDs
  - Option A (default): flux bin method
  - Option B: randomly choose SEDs from the grid
2. Put SEDs on image
  - Option A: spread SEDs across pixels within each SD bin
  - Option B: put SEDs around randomly-selected stars

The `--flux_bin_method` flag was an attempt to distinguish between 1A and 1B.  This is the flag we're trying to rename.  It looks like what you've changed it to is a label for Step 2.  That part doesn't have any input options because the code automatically chooses between 2A and 2B (depending on whether there's a SD/background map listed in datamodel.py).  So I'd propose renaming the new flag to something like `--random_seds`.

Does that make sense?

**Comment by karllark** [comment]
@lea-hagen : yes.  Hopefully the latest changes correct address your comment.  Please review and tell me if this PR is ready for merging yet or needs more work.

File: docs/workflow.rst [hunk 1]
```diff
@@ -120,30 +120,39 @@ The observation model is based on artificial star tests (ASTs).  More details
...
-  * Option 1 (datamodel.ast_source_density_table is set):
-    For each source density or background bin, randomly place the SEDs
-    within pixels of that bin.  Repeat for each of the bins.
-  * Option 2 (datamodel.ast_source_density_table = None):
-    Randomly choose a star from the photometry catalog, and place the
-    artificial star nearby.  Repeat until all SEDs have been placed.
+     $ python -m beast.tools.run.make_ast_inputs --random_seds
 
-.. code-block:: console
+How the sources are placed in the image is determined by the ast.source_density_table
```
Comment (lea-hagen) [inline docs/workflow.rst]: Small typo: `ast.source_density_table` -> `ast_source_density_table`

File: docs/workflow.rst [hunk 2]
```diff
@@ -156,13 +165,13 @@ same criteria must be applied to the AST catalog.
...
+     $ python -m beast.tools.cut_catalogs phot_catalog_with_sourceden.fits phot_catalog_cut.fits \
+           --partial_overlap --region_file --flagged --flag_filter F475W
 
-   $ python -m beast.tools.cut_catalogs phot_catalog_with_sourceden.fits phot_catalog_cut.fits \
-         --partial_overlap --region_file --flagged --flag_filter F475W
-   $
-   $ python -m beast.tools.cut_catalogs ast_catalog.fits ast_catalog_cut.fits \
-         --partial_overlap --region_file --flagged --flag_filter F475W
+     $ python -m beast.tools.cut_catalogs ast_catalog.fits ast_catalog_cut.fits \
+           --partial_overlap --region_file --flagged --flag_filter F475W
```
Comment (lea-hagen) [inline docs/workflow.rst]: I can't actually find what changed in this part

Comment (karllark) [reply docs/workflow.rst]: Just changed the indent so that it matched all the rest of console blocks.

**Pull request merged**